src/Core/InstanceConfigTrait.php

```diff
@@ -169,7 +242,7 @@ protected function _configRead($key)
     }
 
     /**
-     * Write a config variable
+     * Writes a config key.
```

Comment:
This is somewhat misleading as the method allows to write more than one config key by providing an array.